backend/src/main/java/me/performancereservation/domain/refund/RefundService.java

```diff
@@ -0,0 +1,140 @@
+package me.performancereservation.domain.refund;
+
+import jakarta.transaction.Transactional;
+import lombok.RequiredArgsConstructor;
+import me.performancereservation.domain.performance.entities.Performance;
+import me.performancereservation.domain.refund.dto.RefundDetailResponse;
+import me.performancereservation.domain.refund.dto.RefundRequest;
+import me.performancereservation.domain.refund.dto.RefundResponse;
+import me.performancereservation.domain.refund.enums.RefundStatus;
+import me.performancereservation.domain.reservation.Reservation;
+import me.performancereservation.domain.reservation.ReservationRepository;
+import me.performancereservation.domain.reservation.enums.ReservationStatus;
+import me.performancereservation.global.exception.AppException;
+import me.performancereservation.global.exception.ErrorCode;
+import me.performancereservation.global.exception.ErrorType;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Service
+@Transactional
```

Comment:
```suggestion
@Transactional(readOnly = true)
```
@Transactional은 메서드 단위에 붙여 읽기 작업은 최적화 해주는 게 좋을 것 같습니다!